bokeh/models/images.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import
+from ..plot_object import PlotObject
+from ..properties import (Any, Dict, String, Int)
+
+class ImageSource(PlotObject):
+    """ A base class for all image source types. """
+
+    _args = ('url', 'extra_url_vars')
+
+    url = String("url", help="""
+    tile service url (example: http://c.tile.openstreetmap.org/{Z}/{X}/{Y}.png)
+    """)
+
+    extra_url_vars = Dict(String, Any(String, Int), help="""
```

Comment:
> `Any(String, Int)`

This shouldn't be even allowed. `Any` is supposed to not take any type parameters, it's the type above all other types (in Python's terms it's `object`). I presume you just wanted `Dict(String, Any)` (or `Dict(String, PrimitiveProperty)` to be specific, but that may not work)?

btw. This is dynamic typing at its finest. `Any(String, Int)` is equivalent to `Any(default=String, help=Int)`.